pallets/stableswap/src/weights.rs

```diff
@@ -55,81 +60,91 @@ pub trait WeightInfo {
 	fn update_amplification() -> Weight;
 }
 
-pub struct BasiliskWeight<T>(PhantomData<T>);
+pub struct SubstrateWeight<T>(PhantomData<T>);
```

Comment:
How are these weights different from `HydraWeight`?

Reply:
No different. HydraWeights name is used in the runtime. Fort the default, i'd like to have it named SubstrateWeight.